openadapt/models.py

```diff
@@ -323,3 +323,11 @@ class MemoryStat(db.Base):
     recording_timestamp = sa.Column(sa.Integer)
     memory_usage_bytes = sa.Column(ForceFloat)
     timestamp = sa.Column(ForceFloat)
+
+
+class BrowserEvent(db.Base):
+
+    bodyHTML = sa.Column(sa.String)
+    headHTML = sa.Column(sa.String)
+    url = sa.Column(sa.String)
+    timestamp = sa.Column(ForceFloat)
```

Comment:
Please add `recording_timestamp`

Reply:
Addressed in: [1aade8e](https://github.com/OpenAdaptAI/OpenAdapt/pull/364/commits/1aade8e03a15fe9f3944252915454760faac1eac) and [ba220be](https://github.com/OpenAdaptAI/OpenAdapt/pull/364/commits/ba220be409484d871336c71d9b5468a2f1553484)